restore-workspace-tutorial-compiler/setup/setupTutorialCompiler.ps1

```diff
@@ -0,0 +1,8 @@
+npm install -g typescript
+
+git clone https://github.com/devonfw-tutorials/tutorial-compiler.git
+git clone https://github.com/devonfw-tutorials/tutorials.git
+
+cd tutorial-compiler
+npm install 
+cd ..
```

Comment:
Is the command `cd..` needed? The script will be executed in the background, right? So this would have no effect I think